.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+## PR Checklist
+Please check if your PR fulfills the following requirements:
+
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+
```

Comment:
```suggestion
- [ ] In case of non-backward compatible changes. Reviewer should check this item as a write-off, and add details added in #Backward compatibility section
```